### PR TITLE
replaced word complain -> warn

### PR DESCRIPTION
**Pull request by haripriya9647**
Just a word replacement
Issue #43718

### CODE REVIEW

File: content/en/docs/tutorials/security/apparmor.md [hunk 1]
```diff
@@ -70,11 +64,11 @@ Make sure:
    If the Kubelet contains AppArmor support (>= v1.4), it will refuse to run a Pod with AppArmor
    options if the kernel module is not enabled.
 
-  {{< note >}}
-  Ubuntu carries many AppArmor patches that have not been merged into the upstream Linux
-  kernel, including patches that add additional hooks and features. Kubernetes has only been
-  tested with the upstream version, and does not promise support for other features.
-  {{< /note >}}
+{{< note >}}
```
Comment (windsonsea) [inline content/en/docs/tutorials/security/apparmor.md:67]: The original indentation is correct. See the [current online page](https://kubernetes.io/docs/tutorials/security/apparmor/#before-you-begin).

**Review by sftim — COMMENTED** [review]
Thanks for the PR. Why is this the right change to make?

File: content/en/docs/tutorials/security/apparmor.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 reviewers:
-- stclair
+  - stclair
```
Comment (amolmote) [inline content/en/docs/tutorials/security/apparmor.md:3]: dont think this change is required.

File: content/en/docs/tutorials/security/apparmor.md [hunk 3]
```diff
@@ -25,22 +24,16 @@ that AppArmor is not a silver bullet and can only do so much to protect against
 application code. It is important to provide good, restrictive profiles, and harden your
 applications and cluster from other angles as well.
 
-
-
 ## {{% heading "objectives" %}}
 
-
-* See an example of how to load a profile on a node
```
Comment (amolmote) [inline content/en/docs/tutorials/security/apparmor.md:33]: seems same result after the change

**Review by amolmote — COMMENTED** [review]
triage not accepted for this issue. Suggest you to wait for it then you can raise the PR.

Suggest you to look for the potential change, doesn't matter it could be a single character change that will make impact.

**Comment by kbhawkey** [comment]
@haripriya9647 , I don't think the changes are needed.
Do you have more comments or changes, or can you close this pull request?
Thanks

**Pull request closed without merging**